MediaPicker/src/main/java/org/wordpress/android/mediapicker/loader/MediaLoader.kt

```diff
@@ -16,13 +16,9 @@ import org.wordpress.android.mediapicker.loader.MediaSource.MediaLoadingResult
 import org.wordpress.android.mediapicker.loader.MediaSource.MediaLoadingResult.Empty
 import org.wordpress.android.mediapicker.loader.MediaSource.MediaLoadingResult.Failure
 import org.wordpress.android.mediapicker.loader.MediaSource.MediaLoadingResult.Success
-import org.wordpress.android.ui.utils.UiString
-import org.wordpress.android.util.NetworkUtilsWrapper
+import org.wordpress.android.util.UiString
 
-data class MediaLoader(
-    private val mediaSource: MediaSource,
-    private val networkUtilsWrapper: NetworkUtilsWrapper
```

Comment:
`NetworkUtilsWrapper` is never used in [origin implementation](https://github.com/wordpress-mobile/WordPress-Android/blob/70ce7de8817020d29768074bb408688436320220/WordPress/src/main/java/org/wordpress/android/ui/mediapicker/loader/MediaLoader.kt#L26).